packages/url_launcher/url_launcher_web/lib/src/link.dart

```diff
@@ -11,7 +11,8 @@ import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/services.dart';
-
+// used to access the current [UrlStrategy]
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
```

Comment:
I don't think you need to explain why the import exists.

You can do the following to make it clear that you only need one thing from this package:
```suggestion
import 'package:flutter_web_plugins/flutter_web_plugins.dart' show urlStrategy;
```